docs/examples/pam.d/fty

```diff
@@ -36,7 +36,7 @@
 # The 42ity requirements for password:
```

Comment:
Generally LGTM, but I'd suggest to move this comment detailing these requirements into the actual new file that defines them (so they are later changed in sync too). Here you might leave a shorter comment like

````
# The 42ity requirements for password are defined in /etc/pam.d/fty-password 
````